ATM.java

```diff
@@ -0,0 +1,145 @@
+// ATM
+import java.util.Scanner;
+
+public class ATM
+{
+
+    public static void main(String[] args)
+    {
+        Scanner kb = new Scanner(System.in);
+        // Start an ATM Session
+        boolean session = true;
+        // Tracks for interest calculations every 5 transactions
+        int numTransactions = 0;
+        // First user input of D,W,C,T, or Q
+        String choice;
+        SavingsAccount saveAcc = new SavingsAccount(100);
+        CheckingAccount checkAcc = new CheckingAccount(10);
+        System.out.println("Welcome to myATM! How may I assist you?");
+
+        /*
+        TRIED TO ADD A METHOD TO PRINT, BUT COULDN'T GET IT TO WORK
```

Comment:
You have to pass references in to the CheckingAccount and SavingsAccount instances because they are not visible outside of the main method.